cluster-scope/base/external-secrets.io/externalsecrets/open-cluster-management-observability-alertmanager-config/externalsecret.yaml

```diff
@@ -95,6 +102,16 @@ spec:
               text: |{{ .slack_config_text }}
               title: "{{ .slack_config_title }}"
               title_link: "{{ .slack_config_title_link }}"
+          - name: slack-notifications-prod-rhods-ope
+            slack_configs:
+            - channel: '#alerts-prod-rhods-ope'
+              api_url: "{{ .slack_api_url_prod-rhods-ope }}"
```

Comment:
I think all of these references are going to be invalid because `-` isn't allowed in identifiers. Instead of `slack_api_url_prod-rhods-ope` you will need `slack_api_url_prod_rhods-ope` (change in this section and in the `data` section).

Reply:
@larsks this is very awkward :-( a stupid typo, grr-argh, thanks for finding it :+1: